src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/AbstractRazorEditorTest.cs

```diff
@@ -14,7 +14,7 @@
 using Microsoft.Internal.VisualStudio.Shell.Interop;
 using Microsoft.VisualStudio.Settings;
 using Microsoft.VisualStudio.Shell;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
You were using 2 assertions from MSTest but all the rest of the project is using xUnit so I have changed these 2 assertions to use xUnit.